script/Arcueid/Day 13/13_00_ARC13_5.txt

```diff
@@ -449,17 +453,17 @@ But it's not enough. It's nowhere near enough.
 -- Page 429, Offset 32900.
 -- その先に行く。何の躊躇もない。それを<咎|とが>めた女は、もうこの世には存在しない。
 I must go beyond. I mustn't hesitate.
- The woman who warned me of the consequences no longer exists in this world.
+ The woman who warned me of the consequences is not in this world.
```

Comment:
Prefer original

Reply:
I think the original made it seem like "she's already dead" when its like "she's not a part of this world" so I tried rephrasing it to reflect that.

Reply:
Yeah, I mean he _believes_ she is

Reply:
Wait, I thought this was talking about Aoko...

Reply:
Pretty sure this is referring to Arc saying he'll turn into a vegetable if he uses his eyes too much

Reply:
Aoko does give him practically the same advice but on reflection you're probably right in that this interpretation makes more sense given the context of the scene.